data/korath/efreti outfits.txt

```diff
@@ -28,7 +28,7 @@ outfit "Quantum Core (Small)"
 	"hull repair rate" 0.14
 	"hull energy" 0.14
 	"hull heat" 0.21
-	description `The Quarg-inspired modifications to this small systems core diminishes its direct contributions to shield and hull repair, but any ship running extra shield generators will benefit from its optimizing effects.`
+	description `The Quarg-inspired modifications to this small Systems Core means it has less shield regeneration and hull repair than before, but any ship with additional generator or repair units will benefit from its optimizing effects.`
```

Comment:
This is just outright incorrect; the quantum cores only buff shield generators, not hull repair outfits.

Reply:
I misunderstood the attribute definitions.